src/ring_theory/subring.lean

```diff
@@ -1000,11 +1000,11 @@ end actions
 -- both ordered ring structures and submonoids available
 
 /-- The subgroup of positive units of a linear ordered commutative ring. -/
```

Comment:
docs needs updating too ;b

Reply:
Ah jaysus I'm a disaster! Thanks for pointing this out, I'll fix it now.